src/modules/admin/components/formRules/NewFormRuleDialog.tsx

```diff
@@ -245,114 +268,151 @@ const NewFormRuleDialog: React.FC<Props> = ({
       </DialogTitle>
       <DialogContent>
         <Stack gap={2} sx={{ my: 2 }}>
-          {validationError && <Alert severity='error'>{validationError}</Alert>}
+          <Card
+            sx={{ backgroundColor: theme.palette.background.default, p: 2 }}
+          >
+            <BaseFormRule
+              dataCollectedAbout={
+                rule.dataCollectedAbout
+                  ? HmisEnums.DataCollectedAbout[rule.dataCollectedAbout]
+                  : HmisEnums.DataCollectedAbout.ALL_CLIENTS
+              }
+              projectType={
+                rule.projectType ? HmisEnums.ProjectType[rule.projectType] : ''
+              }
+              funder={rule.funder ? HmisEnums.FundingSource[rule.funder] : ''}
+              otherFunder={rule.otherFunder || ''}
+              project={projectName || ''}
+              organization={organizationName || ''}
+              serviceType={serviceTypeName || ''}
+              serviceCategory={serviceCategoryName || ''}
+              formRole={formRole}
+            />
+          </Card>
           {formRole === FormRole.Service && (
             <>
-              <FormRuleSelectField
-                rule={rule}
-                label='Applies to Service Category'
-                name='serviceCategoryId'
-                onChange={(option) => onChangeRule('serviceCategoryId', option)}
-                options={pickListMap.serviceCategoryId}
-              />
-              <FormRuleSelectField
-                rule={rule}
-                label='Applies to Service Type'
-                name='serviceTypeId'
-                onChange={(option) => onChangeRule('serviceTypeId', option)}
-                options={pickListMap.serviceTypeId}
+              <FormRuleCondition
```

Comment:
A design suggestion:
- wrap this in a Box and move it below the data collected about, so it looks more cohesive with the rest of the rules:
<img width="930" alt="Screenshot 2024-08-09 at 10 56 30 AM" src="https://github.com/user-attachments/assets/44cb64ad-c457-4e97-8648-88ad6742d91f">


```
              <Box
                sx={{
                  backgroundColor: theme.palette.grey[100],
                  borderRadius: 1,
                  py: 1,
                  px: 2,
                }}
              >
```

Reply:
I like the idea of adding a box around it. One argument for keeping it above the "Applies to client type" dropdown is that this ordering matches the order of prose when we display the completed rule, both in the top box here on this modal, and on the Rules table outside of this modal. What do you think about this change - adding the box but keeping the order the same?

<img width="865" alt="Screenshot 2024-08-09 at 12 06 55 PM" src="https://github.com/user-attachments/assets/88b70725-4f6c-4b18-9f3f-1ed8e1aeff68">